src/workloads/query/multiplanner/BlockingSort.yml

```diff
@@ -0,0 +1,404 @@
+      - keys: {x17: 1}
+        options: {name: index17}
+      - keys: {x18: 1}
+        options: {name: index18}
+      - keys: {x19: 1}
+        options: {name: index19}
+      - keys: {x20: 1}
+        options: {name: index20}
+      - keys: {x21: 1}
+        options: {name: index21}
+      - keys: {x22: 1}
+        options: {name: index22}
+      - keys: {x23: 1}
+        options: {name: index23}
+      - keys: {x24: 1}
+        options: {name: index24}
+      - keys: {x25: 1}
+        options: {name: index25}
+      - keys: {x26: 1}
+        options: {name: index26}
+      - keys: {x27: 1}
+        options: {name: index27}
+      - keys: {x28: 1}
+        options: {name: index28}
+      - keys: {x29: 1}
+        options: {name: index29}
+      - keys: {x30: 1}
+        options: {name: index30}
+      - keys: {x31: 1}
+        options: {name: index31}
+      - keys: {x32: 1}
+        options: {name: index32}
+      - keys: {x33: 1}
+        options: {name: index33}
+      - keys: {x34: 1}
+        options: {name: index34}
+      - keys: {x35: 1}
+        options: {name: index35}
+      - keys: {x36: 1}
+        options: {name: index36}
+      - keys: {x37: 1}
+        options: {name: index37}
+      - keys: {x38: 1}
+        options: {name: index38}
+      - keys: {x39: 1}
+        options: {name: index39}
+      - keys: {x40: 1}
+        options: {name: index40}
+      - keys: {x41: 1}
+        options: {name: index41}
+      - keys: {x42: 1}
+        options: {name: index42}
+      - keys: {x43: 1}
+        options: {name: index43}
+      - keys: {x44: 1}
+        options: {name: index44}
+      - keys: {x45: 1}
+        options: {name: index45}
+      - keys: {x46: 1}
+        options: {name: index46}
+      - keys: {x47: 1}
+        options: {name: index47}
+      - keys: {x48: 1}
+        options: {name: index48}
+      - keys: {x49: 1}
+        options: {name: index49}
+      - keys: {x50: 1}
+        options: {name: index50}
+      - keys: {x51: 1}
+        options: {name: index51}
+      - keys: {x52: 1}
+        options: {name: index52}
+      - keys: {x53: 1}
+        options: {name: index53}
+      - keys: {x54: 1}
+        options: {name: index54}
+      - keys: {x55: 1}
+        options: {name: index55}
+      - keys: {x56: 1}
+        options: {name: index56}
+      - keys: {x57: 1}
+        options: {name: index57}
+      - keys: {x58: 1}
+        options: {name: index58}
+      - keys: {x59: 1}
+        options: {name: index59}
+      - keys: {x60: 1}
+        options: {name: index60}
+      - keys: {x61: 1}
+        options: {name: index61}
+      - keys: {x62: 1}
+        options: {name: index62}
+      - keys: {x63: 1}
+        options: {name: index63}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+
+- Name: SimpleMultiplannerWith64Indexes
```

Comment:
Change the name of the actors. These are the same as Simple.yml. I suggest something like `MultiplannerWith64IndexesBlockingSort`. Same for other actor names.